tools/specgen/specgen.go

```diff
@@ -38,7 +38,6 @@ type typeDef struct {
 var dockerTypesToReflect = []typeDef{
 
 	// POST /auth
-	{reflect.TypeOf(AuthConfigParameters{}), "AuthConfigParameters"},
```

Comment:
Its safe to leave this but say types.AuthConfig{} that way its explicitly included if they every change how the types are "inherited"